test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs

```diff
@@ -234,42 +244,71 @@ defmodule Membrane.HTTPAdaptiveStream.SinkBinIntegrationTest do
       |> Enum.each(fn manifest_filename ->
         manifest_file = File.read!(Path.join(tmp_dir, manifest_filename))
 
-        segments_in_manifest =
-          Regex.scan(~r/#EXTINF:\d+\.\d+,\s\w+segment_\d+_.+.m4s/, manifest_file)
-
-        number_of_segments_in_manifest = Enum.count(segments_in_manifest)
-        # delta manifest will be generated when plailist is longer then 6 segments
-        if number_of_segments_in_manifest > @expected_number_of_segments_in_delta_playlist do
-          # check if manifest contains CAN-SKIP-UNTIL tag
-          assert Regex.match?(~r/CAN-SKIP-UNTIL=\d+\.*\d*/, manifest_file)
-
-          # check if delta file exists
-          delta_manifest_filename = String.replace(manifest_filename, ".m3u8", "_delta.m3u8")
-          assert File.exists?(Path.join(tmp_dir, delta_manifest_filename))
-
-          delta_manifest_file = File.read!(Path.join(tmp_dir, delta_manifest_filename))
-
-          segments_in_delta_manifest =
-            Regex.scan(~r/#EXTINF:\d+\.\d+,\s\w+segment_\d+_.+.m4s/, delta_manifest_file)
-
-          number_of_segments_in_delta_manifest = Enum.count(segments_in_delta_manifest)
-          # check if delta manifest contains exected number of segments
-          assert number_of_segments_in_delta_manifest ==
-                   @expected_number_of_segments_in_delta_playlist
-
-          # check if delta manifest contains last 6 segments from manifest
-          assert Enum.take(segments_in_manifest, -@expected_number_of_segments_in_delta_playlist) ==
-                   segments_in_delta_manifest
-
-          # check if delta manifest contains #EXT-X-SKIP tag with correct value
-          [_match, skipped_segments] =
-            Regex.run(~r/EXT-X-SKIP:SKIPPED-SEGMENTS=(\d+)/, delta_manifest_file)
-
-          {skipped_segments, _rest} = Integer.parse(skipped_segments)
-
-          assert skipped_segments + number_of_segments_in_delta_manifest ==
-                   number_of_segments_in_manifest
-        end
+        assert target_duration =
```

Comment:
I don't know if assertion should be used like that. @mat-hek?

Reply:
This assertion always passes, so it doesn't make sense IMO 🤔

Reply:
Changed all occurences